### PR TITLE
Update docs owned by Research Team

### PR DESCRIPTION
**Pull request by lukevalenta**
Update distributed-web, randomness-beacon, and time-services developer docs

### CODE REVIEW

File: products/access/src/content/about/how-access-works.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 order:2
 ---
 
-# How Access Workers
+# How Access Works
```
Comment (wbl) [inline products/access/src/content/about/how-access-works.md:5]: Looks spurious

File: products/time-services/src/content/nts/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+Network Time Security (NTS) provides cryptographic security for the client-server mode of the Network Time Protocol (NTP). This enables users to obtain time in an authenticated manner. 
 
+The NTS protocol is divided into two-phases. The first phase is the NTS key exchange that establishes the necessary key material between the NTP client and the server. This phase uses the Transport Layer Security (TLS) handshake and relies on the same public key infrastructure as the web. Once the keys are exchanged, the TLS channel is closed and the protocol enters the second phase. In this phase the results of that TLS handshake are used to authenticate NTP time synchronization packets via extension fields. For more information, read the [Internet draft](https://tools.ietf.org/html/draft-ietf-ntp-using-nts-for-ntp-19).
```
Comment (wbl) [inline products/time-services/src/content/nts/index.md:9]: I should have changed this to be RFC 8915 already https://tools.ietf.org/html/rfc8915

File: products/time-services/src/content/nts/usage.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
-weight: 10
+title: User Guide
+order: 0
 ---
 
+# Using Cloudflare's Secure Time Service
+
 ## NTS Client
 
 You can use time.cloudflare.com as the source of time for all your devices today with NTP, while NTS clients are still under development.
```
Comment (wbl) [inline products/time-services/src/content/nts/usage.md:10]: Chrony now has support, so maybe we need to change this.

**Review by wbl — APPROVED** [review]
Looks good, but two or three little nits I noticed.

File: products/access/src/content/about/how-access-works.md (hunk 1 above)
Comment (ltv511) [reply products/access/src/content/about/how-access-works.md:5]: Yeah, I'll remove this and instead ping the Access team to fix the typo 👍

**Comment by adamschwartz** [comment]
This looks really rad. Thank you!

**Pull request merged**